.github/workflows/test_requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy>=1.14,<1.18
 scipy>=1.5.2
-torch>=1.6.0
+torch==1.7.0
```

Comment:
Was this change intended?

Reply:
Not sure. But the symptom was that the CI did not successfully run. I think it was due to torch 1.7.1 (the 700 MB variant) being download instead of the 1.7.0 CPU one (~60 MB), which we ran in tests earlier.